File: array_key_exists/array_key_exists.py
# ATOMIC_FUNCTION: array_key_exists (key,dic) 
# TAGS: dictionary
# LANG:  python
# FUNCTION: array_key_exists (key,dic) 
def array_key_exists (key,dic) :
	return key in dic

start_val_1 = {"first":1}
start_val_2 = "first"
expected_end_val = True

end_val = array_key_exists (start_val_2,start_val_1) 

# TEST
if(end_val == expected_end_val):
    print("PASS")
else:
    print("FAIL")

File: array_key_exists/array_key_exists.js
// ATOMIC_FUNCTION: array_key_exists(k,m)
// TAGS: map
// LANG:  node.js
// FUNCTION: array_key_exists(k,m)
function array_key_exists(k,m) {
   return m.has(k);
}
var start_val_1 = new Map();
start_val_1.set("first",1)
var start_val_2 = "first";
var expected_end_val = true;
var end_val = array_key_exists(start_val_2,start_val_1)

// TEST
if(end_val==expected_end_val){
    console.log('PASS');
}else{
	
    console.log('FAIL');
}
